src/lib/timeseries.js

```diff
@@ -57,6 +57,10 @@ export const transformSerieToTrip = serie => {
 
 export const transformTimeseriesToTrips = timeseries => {
   return timeseries.flatMap((timeserie, index) => {
+    if (!timeserie.series) {
+      throw `The timeserie with id ${timeserie._id} must have a "series" attributes to make transformTimeseriesToTrips() working properly`
+    }
```

Comment:
https://web.archive.org/web/20111226063046/http://www.devthought.com/2011/12/22/a-string-is-not-an-error/

While you can throw a string, it's considered as not a good practice. There is a lot of place where you'll not have the stacktrace, message or else (chrome devtools has this capability, I don't know if node or wkwebview had it for instance). We should always `throw a new Error()` ;)

Reply:
ok merci 👍